usaspending_api/conftest.py

```diff
@@ -588,3 +588,21 @@ def fake_sqs_queue():
     q.purge()
     q.reset_instance_state()
     remove_unittest_queue_data_files(q)
+
+
+@pytest.fixture
+def elasticsearch_recipient_index(db):
```

Comment:
Tests look good. I'd maybe just move this up with the other ES fixtures.

Reply:
Just pushed a commit with the new fixture moved up with the other ES fixtures.